configs/mconfig.Linux

```diff
@@ -38,6 +38,9 @@ TEST_LDFLAGS=$(TEST_LDFLAGS_BASE) $(TEST_CXXFLAGS)
 # Features.
 
 SUPPORT_CGROUPS=1
+SUPPORT_CAPABILITIES=1
```

Comment:
Given this requires a dependency (in libcap), I'd prefer the default here was `SUPPORT_CAPABILITIES=0`. It's fine to have it on-by-default in the `configure` script since we can check the dependency is available there (but that should be implemented).

Reply:
it may add a dependency, but it's a dependency that's present on virtually *any* linux system (it's required by things like... udev), imo it makes much more sense to keep it on

Reply:
The library will be present but note necessarily the headers, and people using a straight Makefile-based build are often just trying out Dinit for personal use. I'd like to make sure that this scenario still works as easily as possible.

Reply:
yes, but those people can easily disable it; i don't think it's right to default to a configuration where capabilities are not supported, the default build should imo be one that is as distros should build it

it's trivial to disable for somebody just trying it (it's also trivial to just install libcap-devel or whatever, in 99% of scenarios it's easily doable, and you already need to install a development toolchain)

Reply:
For this build settings template file I'm not so fussed, but at least `-lcap` should be added to `LDFLAGS_BASE` if `SUPPORT_CAPABILITIES=1` is the default. (Or add a separate LIBCAP variable, set it to `-lcap`, and use it in the Makefiles in appropriate places).

For the `configure`-script though, the whole point of the script is to automatically find build settings that will work on the current system as it is.

Reply:
I think I can implement a library finding function based on `pkg-config` for `configure` script.

Reply:
I misunderstood what you meant the first time. I changed the default mconfigs to disable capabilities, and the configure script detects -lcap for base ldflags and uses it (which is also enough to address the rest)

Since most people are supposed to run configure, I think that's okay